protos/flyteidl/admin/node_execution.proto

```diff
@@ -171,15 +185,17 @@ message NodeExecutionGetDataRequest {
 // Response structure for NodeExecutionGetDataRequest which contains inputs and outputs for a node execution.
 message NodeExecutionGetDataResponse {
     // Signed url to fetch a core.LiteralMap of node execution inputs.
-    UrlBlob inputs                             = 1;
+    // Deprecated: Please use full_inputs instead.
+    UrlBlob inputs                             = 1  [deprecated = true];
 
     // Signed url to fetch a core.LiteralMap of node execution outputs.
-    UrlBlob outputs                            = 2;
+    // Deprecated: Please use full_outputs instead.
+    UrlBlob outputs                            = 2  [deprecated = true];
 
-    // Optional, full_inputs will only be populated if they are under a configured size threshold.
+    // Full_inputs will only be populated if they are under a configured size threshold.
```

Comment:
what if they are over the size threshold? I think that will cause runtime failures right?

Reply:
@kumare3 perhaps, the input and output URIs are always available in the closure

what do you think is right here? always return inline and remove the size check altogether?